_static/css/custom.css

```diff
@@ -1,4 +1,6 @@
-.wy-nav-content { 
+@import url("https://cdn.cyberduck.io/css/custom.css");
```

Comment:
Is there a reason to import the stylesheet from an external resource?

Reply:
Only needs to be maintained once.